### PR TITLE
SAK-40742 Assignment further attempt to eliminate duplicate submissions

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java [hunk 1]
```diff
@@ -1913,7 +1915,7 @@ public boolean canSubmit(Assignment a) {
     }
 
     @Override
-    @Transactional
+    // @Transactional(isolation = Isolation.SERIALIZABLE)
```
Comment (ottenhoff) [inline assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java]: is this commented out purposefully?

**Comment by ern** [comment]
I will have another update to this to further clean it up.

File: assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java (hunk 1 above)
Comment (ern) [reply assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java]: yeah part of the clean up i will need to do

**Comment by ottenhoff** [comment]
This is becoming essential for 12.x ..... I think we should get this merged in after some quick review.

**Pull request merged**